dataframe/edm4hepRDF.cc

```diff
@@ -0,0 +1,42 @@
+#include "edm4hepRDF.h"
+#include "Math/Vector4D.h"
+
+namespace edm4hep {
+
+std::vector<float> pt (std::vector<MCParticleData> in){
```

Comment:
```suggestion
std::vector<float> pt (std::vector<MCParticleData> const& in){
```

Reply:
Agreed, I'll add that to all the methods.